c10/core/DispatchKey.h

```diff
@@ -135,7 +135,7 @@ enum class DispatchKey : uint16_t {
   //
   // and Undefined == nullopt.  We didn't actually represent
   // it this way because optional<RealDispatchKey> would take two
-  // words, when DispatchKey fits in eight bits.
```

Comment:
8 is probably correct here from reading the sentence: even when DispatchKey fits in 8 bits, the optional<RealDispatchKey> needs 2 words.

Reply:
The datatype of `DispatchKey` is `uint16_t`, so maybe 16 bits is correct? (See line 128 of file c10/core/DispatchKey.h)